### ISSUE TITLE
Seems to truncate line and fail to compile SCSS

### ISSUE THREAD

**Issue by luizfb** [sass/libsass#1146]
I am using the latest version of CodeKit which includes LibSass 3.2.

In a very short file that only includes all of my other ones, I have a line that imports some fonts from Google Fonts:
```
@import url(http://fonts.googleapis.com/css?family=Karla:400,700,400italic|Anonymous+Pro:400,700,400italic);
```

It seems to be truncating the line and then accuses it of not having a closing `)`. Everything compiles fine with the Ruby compiler and had been compiling fine with LibSass for months.

![screen shot 2015-04-29 at 3 48 17 pm](https://cloud.githubusercontent.com/assets/1904278/7398902/600fd33c-ee88-11e4-93e4-332b9a4d9a0e.png)

**Comment by Snugug** [comment]
Please 2x check that it's using Libsass 3.2. Libsass 3.2 was only released 2 days ago, so unless there was an update today you may be using an older version

**Comment by Snugug** [comment]
Duplicate of #1145

**Comment by xzyfer** [comment]
Fixed in [3.2.1](https://github.com/sass/libsass/releases/tag/3.2.1)
